README.md

```diff
@@ -46,234 +46,59 @@ Git Proxy deploys custom push protections and policies on top of Git. It is a hi
 Git Proxy is built with a developer-first mindset. By presenting simple-to-follow remediation instructions in the CLI/Terminal, it minimises the friction of use and adoption, and keeps developers focused on what matters; committing and pushing code.
 
 
-## Installation
+## Getting Started 🚀
 
-To install Git Proxy, you must first install [Node.js](https://nodejs.org/en/download). Then, use the [npm](https://www.npmjs.com/) package manager:
+Install & run git-proxy (requires [Nodejs](https://nodejs.org/en/download/)):
 
 ```bash
-npm install -g @finos/git-proxy
+$ npx -- @finos/git-proxy
+Need to install the following packages:
+  @finos/git-proxy@1.1.0
+Ok to proceed? (y) 
+...
+creating user
+        user=admin,
+        gitAccount=none
+        email=admin@place.com,
+        admin=true
+Service Listening on 8080
+Listening on 8000
 ```
 
-To install a specific version of Git Proxy, append the version to the end of the `install` command:
+Clone a repository, set the remote to the Git Proxy URL and push your changes:
 
 ```bash
-npm install -g @finos/git-proxy@1.1.0
+$ git clone https://github.com/octocat/Hello-World.git && cd Hello-World
+$ gh repo fork
+✓ Created fork yourGithubUser/Hello-World
+...
+$ git remote set-url origin http://localhost:8000/yourGithubUser/Hello-World.git
+$ git push
+Enumerating objects: 4, done.
+Counting objects: 100% (4/4), done.
+Delta compression using up to 8 threads
+Compressing objects: 100% (2/2), done.
+Writing objects: 100% (3/3), 946 bytes | 946.00 KiB/s, done.
+Total 3 (delta 0), reused 0 (delta 0), pack-reused 0
+remote: ERR     Rejecting repo yourGithubUser/Hello-World.git not in the authorisedList
```

Comment:
Does it make sense to show a failed attempt in the `README.md` instead of an intercepted one?

Reply:
See https://github.com/finos/git-proxy/pull/327#discussion_r1375265790 - I'm happy to remove this completely since the user journey is captured in the quick start guide.

Reply:
@coopernetes - just scrolled down to this comment. I'd agree, let's just show running Git Proxy and point the developer to the docs 👍

Reply:
Did you want me to remove the failing push output from the README? I already included a note describing that you should expect a push to fail without modifying the default configuration, which I believe is important to state right up front. A more complete, end-to-end example is in documentation and makes sense to direct new users there.

Reply:
Should we just remove it entirely, as you suggested earlier and use more of the estate to point the viewer to the documentation site instead? @coopernetes